common/core/desktop/.gitignore

```diff
@@ -9,3 +9,5 @@ wasm.build
 TIER.md
 VERSION.md
 shellHelperFunctions.sh
+
+src/rust/keyman_keyboard_processor/target/
```

Comment:
This shouldn't be necessary. Build files should be going into build/. If not, I need to fix that.

Reply:
One of the VSCode extensions which runs in the background puts its files there.

Reply:
done